david-fraley/MassVaccinationSystem
# MVS-722 adjust background color of left bar/tab based on dose in pod app

## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira


## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-722

## :newspaper: [Work Description]
Implemented a colorblind-friendly color scheme to indicate how many doses a patient has received / what dose they will receive.  The "Retrieve Patient Record" and (new) "Information" pages are exempt from this color-coding scheme.
Added an "Information" page at the bottom of the left menu to provide information on the color-coding scheme.  The thought is that this general "Information" page could become a helpful/central location for user information.

## :vertical_traffic_light: [Testing/QA Notes]
The following test patients were created for testing purposes (please feel free to use them!)
- Patient, No (DOB: 01/01/2000):  no doses have been administered
- Patient, One (DOB: 01/01/2000):  one dose has been administered
- Patient, Two (DOB: 01/01/2000):  two doses have been administered

Recommended testing/test points:
- Verify that the "Retrieve Patient Record" page is not color-coded (presented in grayscale colors) at any time during testing.
- Verify that the "Information" page is not color-coded (no background color) at any time during testing.
- Register a new patient via the patient registration app.
- When your new patient is retrieved and loaded in the POD app, verify that the blue color scheme is applied to the background and page title bar of the Check-In, Consent and Screening, Vaccination Event, and Discharge pages as you complete the workflow to administer the first dose of a vaccine.
- Go back to the "Retrieve Patient Record" page and retrieve the patient record again.
- Verify that the orange/yellow color scheme is applied to the background and page title bar of the Check-In, Consent and Screening, Vaccination Event, and Discharge pages as you complete the workflow to administer the second dose of a vaccine.
- Go back to the "Retrieve Patient Record" page and retrieve the patient record again.
- Verify that the red/megenta color scheme is applied to the background and page title bar of the Check-In, Consent and Screening, Vaccination Event, and Discharge pages as you complete the workflow to administer a booster dose of a vaccine.
-  Go back to the "Retrieve Patient Record" page and retrieve the patient record again.
- Verify that the red/megenta color scheme is applied (this is applied for patients who have received 2 or more doses)
